halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/GateService.java

```diff
@@ -90,14 +93,19 @@ public List<Profile> getProfiles(
   private GateProfileFactory getGateProfileFactory(String deploymentName) {
     String version =
         getArtifactService().getArtifactVersion(deploymentName, SpinnakerArtifact.GATE);
+    log.info("the current spinnaker version is: " + version);
     try {
       if (Versions.lessThan(version, BOOT_UPGRADED_VERSION)) {
         return boot128ProfileFactory;
       }
+
+      if (Versions.lessThan(version, "6.67.0")) {
```

Comment:
Please add a comment explaining why we're doing this.

Reply:
done

https://github.com/spinnaker/halyard/pull/2216/commits/47ec3c88164d1bd537a1a0483d30244289cb874e